pkgs/racket-benchmarks/tests/racket/benchmarks/shootout/spectralnorm-par.rkt

```diff
@@ -4,11 +4,8 @@
 ;; Translated from Mike Pall's Lua version.
 ;; Parallelized by Sam Tobin-Hochstadt
 
-(require racket/cmdline racket/future
-         racket/require (for-syntax racket/base)
-         (filtered-in (λ (name) (regexp-replace #rx"unsafe-" name ""))
-                      racket/unsafe/ops)
-         (only-in racket/flonum make-flvector))
+(require racket/cmdline racket/future racket/fixnum racket/flonum future-visualizer)
```

Comment:
Is the change to require `future-visualizer` intended?

Reply:
Whoops, fixed.